ci/build-rosette.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+source ci/header.sh
+
+sudo apt install g++ g++-multilib cmake
+cd "${rosette_root_dir}"; ./build.sh
```

Comment:
I think that Henry's new `nix` script means that this should be `nix-build` instead? @henrytill ?

Reply:
I certainly think it's worth doing.  I can provide detailed instruction after doing some tests.

Testing happening [here](https://github.com/rchain/rchain/pull/464).

Reply:
Good news: I've got tested and working example of doing a Rosette Nix build over there at https://github.com/rchain/rchain/pull/464.

@jeremybusk - It's important that you set up the build matrix in a similar fashion.  Also notice the changes I've made to `run.sh` to execute the rosette binary built by Nix.

 Hopefully this should give you a good start with that work.

(Also the `build-subprojects.sh` is just crying out for a case statement!  Check out what I did over there)